pkg/cmd/tknpac/info/globbing_test.go

```diff
@@ -0,0 +1,65 @@
+package info
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	tknpactest "github.com/openshift-pipelines/pipelines-as-code/test/pkg/cli"
+	"gotest.tools/v3/assert"
+	"gotest.tools/v3/fs"
+	"gotest.tools/v3/golden"
+)
+
+func TestGlobbing(t *testing.T) {
+	tests := []struct {
+		name    string
+		pattern string
+		wantErr bool
```

Comment:
can you add few more scenario which covers `wantErr` as well

Reply:
good catch, thanks for the review, i have addressed it